### PR TITLE
chore: update storybook version, add props descriptions

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-592.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-592.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-592.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-592.gearboxjs.preview.cogniteapp.com

File: src/@types/griff-react/index.d.ts [hunk 1]
```diff
@@ -46,13 +43,23 @@ declare module '@cognite/griff-react' {
...
     pointsPerSeries: number;
-    series: any;
+    series: DataProviderSerie[];
     timeDomain: number[] | Domain;
     updateInterval: number;
     collections: any;
   }
 
-  export class DataProvider extends React.Component<DataProviderProps> {}
+  export interface DataProviderSerie {
```
Comment (nalaka) [inline src/@types/griff-react/index.d.ts]: Typo: probably should change to `DataProviderSeries`

**Comment by nalaka** [comment]
(1) Refactor in `preview.js` requested above.

(2) Suggestion on adding a comment to empty `manager.js` file

(3)  Has replaced `config.js`, `addons.js` with `main.js` 

... **Ok**

(4) Follow recommendation to use `/` as the hierarchy separator

... **Ok**

(5) Strategy for getting `<Props of={FooBar} />` to render a property table in Storybook 

In each story file (`components/FooBar/stories/basic.story.tsx`) you have:

```tsx
import { ComponentProps } from './helper';

<Props of={ComponentProps} />
```

Where `ComponentProps` is defined as:

```tsx
export const ComponentProps: FC<FooBarProps> = () => <></>;
```

Isn't it possible to use a pattern like below directly now that we have moved the definition of FooBarProps to the corresponding `interfaces.ts` file?

```jsx
import { FooBarProps } from './interfaces';
<Props of={FooBarProps} />
```

... **Need clarification**

References:

1. [Announcement on the config change](https://medium.com/storybookjs/declarative-storybook-configuration-49912f77b78)
2. [Storybook docs for v5.3](https://storybook.js.org/docs/basics/introduction/)

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import { addParameters, addDecorator } from '@storybook/react';
+import { DocsContainer, Preview, SourceState } from '@storybook/addon-docs/dist/blocks';
+
+const Decorator = (fn) => (
+  <Wrapper>
```
[the remark this answers is not in the document]
Comment (mrtraser) [reply .storybook/preview.js]: Not exactly, cause in this case wrapper defines area where content will be centered and limits contents with to `1000px`, like in addon docs to keep the same styling.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-592.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Unfortunately, we can't set `Docs` tab as default tab (https://github.com/storybookjs/storybook/issues/7066). We will configure this right after storybook provides option for tab configuration.

**Pull request merged**